largescale/many-collection-test.py

```diff
@@ -492,11 +524,12 @@ def populate_collections(num_collections, docs_per, num_workers):
     time_secs = t2-t1
     print_msg("populate_collections", 0, "Done. %d workers took %.2f seconds. Inserts per sec: %d." % (num_workers, time_secs, int(working_set_docs/time_secs)))
 
+# MongoDB setup by initiating a replica set.
 def setup_mongodb(config):
```

Comment:
The logic to populate `if populate and oplog:` has been moved to the main section.